tests/test_bgp_neighbor_af.rb

```diff
@@ -66,9 +68,9 @@ def clean_af(af_args, ebgp=true)
     # 3  => [1, 'default', '10:1::1', %w(ipv6 multicast)],
     # 4  => [1, 'default', '10:1::1', %w(ipv6 unicast)],
     # 5  => [1, 'default', '1.1.1.1', %w(ipv4 multicast)],
-    6 => [1, 'default', '1.1.1.1', %w(ipv4 unicast)],
+    6  => [1, 'default', '1.1.1.1', %w(ipv4 unicast)],
```

Comment:
Yes :-) 

I did that. It aligns the hash values with the keys that have double-digits.